### PR TITLE
Ch setup codeclimate 163496103

### PR DESCRIPTION
**Pull request by kenzdozz**
#### What does this PR do?
Integrated codeclimate maintainability
#### Description of Task to be completed?
Added triggers on travis.yml to run coverage 
Linked repository to codeclimate
Added codeclimate maintainability badge to readme
#### How should this be manually tested?
Check the readme for maintainability badge from codeclimate
#### What are the relevant pivotal tracker stories?
#163496103

### CODE REVIEW

**Comment by ascii-dev** [comment]
Hi @kenzdozz, any reason why you have 10 commits for this task. I think you should combine them into one commit. Thanks

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,17 @@
 language: node_js
+
 node_js:
   - "node"
-after_success: npm run coveralls
+
+# before_script:
+#   - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+#   - chmod +x ./cc-test-reporter
+#   - ./cc-test-reporter before-build
```
Comment (ascii-dev) [inline .travis.yml]: Any reason for pushing comments? Thanks

File: .travis.yml [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+
+cache:
+  directories:
+    - node_modules
+
+after_script:
+  - npm run coveralls
+  # - ./cc-test-reporter format-coverage -p coverage/lcov.info
```
Comment (ascii-dev) [inline .travis.yml]: Same here as above. Thanks

**Comment by kenzdozz** [comment]
I was trying out codeclimate test coverage. I will remove the comments. Thanks

**Comment by kenzdozz** [comment]
> Hi @kenzdozz, any reason why you have 10 commits for this task. I think you should combine them into one commit. Thanks

I have combined the commits to one

File: .travis.yml (hunk 2 above)
Comment (kenzdozz) [reply .travis.yml]: Comments now removed.

**Review by ascii-dev — APPROVED** [review]
LGTM

**Pull request merged**